docs/source/day-to-day.rst

```diff
@@ -82,7 +82,7 @@ Step two: Setup authentication
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Next decide how you'd like to authenticate your hub. You can use Github,
 Google or a "hash" based authenticator. Read more about that here
-`Read more about that here <https://earthlab-hub-ops.readthedocs.io/en/latest/>`_
+`Read more about that here <https://earthlab-hub-ops.readthedocs.io/en/latest/authentication.html>`_
```

Comment:
We should find a way to do this without having to have the explicit URL here, something to fix in a new PR though, let's not hold this one up.

Reply:
cool... open to any ideas on that!